vnext/Microsoft.ProjectReunion.ReactNative/Microsoft.ProjectReunion.ReactNative.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.ProjectReunion.ReactNative</id>
```

Comment:
Any idea what the long term naming is for reunion?  Or how any other libraries are naming themselves to differentiate between reunion and system?

Or should it be another target framework / directory in the existing nuget?

Reply:
nothing to discuss here in public :)